src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private int index = new Random().nextInt(Color.values().length);
```

Comment:
you should make Random variable global, not index, an index should be generated inside of getRandomColor()